app.js

```diff
@@ -58,7 +52,68 @@ var itemObjects = [
 ];
 
 var itemsObjectsWorking;
-var itemsChosen = 0;
+var itemsChosen = window.localStorage.itemsChosen || 0;
+var itemsDisplayed = window.localStorage.itemsDisplayed || 0;
+
+function onLoadValues(){
+  updateItemObjects();
+  if(itemsChosen > 24){
+    var surveyButtonLocation = document.getElementById('startSurvey');
+    surveyButtonLocation.setAttribute('style', 'display:none');
+    buildAPrettyChart();
+  }
+  else if(itemsDisplayed > 0){
+    var surveyButtonLocation = document.getElementById('startSurvey');
+    surveyButtonLocation.setAttribute('style', 'display:none');
+    displayPreviousObjects();
+  }
+}
+
+function updateItemObjects(){
+  if(window.localStorage.itemObjectSavedVallues){
+    var storedObjects = JSON.parse(window.localStorage.itemObjectSavedVallues);
+    itemObjects.map(function(item){
+      item.numberOfTimesShown = storedObjects[item.itemName].numberOfTimesShown;
+      item.usedInLastItemSet = storedObjects[item.itemName].usedInLastItemSet;
+      item.numberOfClicks = storedObjects[item.itemName].numberOfClicks;
+    });
+  }
+}
+
+function displayPreviousObjects(){
+  var locations = ['surveyFirstItem','surveySecondItem','surveyThirdItem'];
+  var lastDisplayed = itemObjects.filter(function(item){
+    if(item.usedInLastItemSet === true){
+      return true;
+    }
+  });
+  console.log(lastDisplayed);
+  lastDisplayed.map(function(item,index){
```

Comment:
I don't remember if I mentioned it in class or not, but good use of the `index` argument here.